src/main/java/seedu/todo/commons/util/FxViewUtil.java

```diff
@@ -20,6 +21,7 @@ public static void applyAnchorBoundaryParameters(Node node, double left, double
     
     /**
      * Hides a specified UI element, and ensures that it does not occupy any space.
+     * @author Wang Xien Dong
```

Comment:
This isn't the correct syntax, if your intention is to mark your code for collation - it should be `@@author` and your matric number instead. You also do not need to mark every method in that case - just put that in front of all the lines you wrote